src/dbnode/storage/bootstrap/bootstrapper/commitlog/source_prop_test.go

```diff
@@ -40,7 +40,9 @@ import (
 	"github.com/m3db/m3/src/dbnode/persist/fs/commitlog"
 	"github.com/m3db/m3/src/dbnode/storage/bootstrap/result"
 	"github.com/m3db/m3/src/dbnode/storage/namespace"
+	topotestutils "github.com/m3db/m3/src/dbnode/topology/testutil"
```

Comment:
same as other spot, `tu` would probably be more concise

Reply:
done